include/seqan3/argument_parser/validators.hpp

```diff
@@ -595,9 +595,9 @@ class output_file_validator : public file_validator_base
     //!\brief Returns a message that can be appended to the (positional) options help page info.
     std::string get_help_page_message() const
     {
-        return detail::to_string("Valid output file formats: [",
-                                 extensions | std::views::join(std::string{", "}),
-                                 "]");
+        using namespace std::literals::string_literals;
```

Comment:
what happend with to_string?

Reply:
It causes an ICE because of a ranges-v3 but that is fixed if we update to the new ranges release, but since this is stalled I wanted to get this PR ready to merge anyway. I hope the ranges update comes soon but since this PR is of interest to users we should discuss to merge it anyway if we cannot update soonish.

Reply:
Pheww, I know what you mean, but this will then stay in this forever. Unless, we can employ something that I just read about recently: https://www.fluentcpp.com/2019/01/01/todo_before-clean-codebase-2019/
The idea is to have a time based static assert, that triggers an assertion after a certain date has passed. Thus we could say ok in 2 weeks I reevaluate it again. But that's just gadget-me talking.

Reply:
Pheww this is hacky :laughing: 
I kind of hoped then we pull the ranges update before this review is done. But the change is not too bad is it? I mean it is just another way of solving this, not a real workaround..

Reply:
seems like #1267 can soon be merged